pkg/controllers/vdb/config.go

```diff
@@ -115,6 +117,19 @@ func (g *ConfigParamsGenerator) GetConfigParms() *vtypes.CiMap {
 	return g.ConfigurationParams
 }
 
+// setup will initialize parms in the ConfigParamsGenerator struct
+func (g *ConfigParamsGenerator) setup() error {
+	if g.ConfigurationParams == nil {
+		g.ConfigurationParams = vtypes.MakeCiMap()
+	}
+	var ok bool
+	g.VInf, ok = g.Vdb.MakeVersionInfo()
```

Comment:
I thought we already checked this on `createdb_reconciler`.

Reply:
We use `ConfigParamsGenerator` outside of create db. So, it is best not to rely on that.